blog-posts/scoped-registry.md

```diff
@@ -0,0 +1,100 @@
+---
+path: "/blog/scoped-registry/"
+date: "2021-12-30"
+title: "NPM scoped registry and authentication"
+featuredImage: images/npm-scoped-registry.jpg
+attribution:
+    creator: Paul Esch-Laurent
+    source: https://unsplash.com/photos/oZMUrWFHOB4
+seoMetaText: create scoped packages and scoped registry and configure authentication via npmrc
+teaserText: What are scoped packages, scoped registry and configure authentication via npmrc
+author: Mark Altmann
+authorSummary: dev@satellytes, ❤️ for Wombats
+
+---
+## TL;DR
+
+Packages in npm can be grouped together with scopes. Scopes are preceded by a `@` followed by the scope name and the package name separated by `/` e.g. `@scope/package`. To set a specific registry for a scope you can use scoped registries. To set a scoped registry open up the .npmrc file and enter the following:
+
+`@scope:registry=https://<your registry>` where `@scope` is your scope name and `<your registry>` is the registry where your scoped packages are hosted. 
+
+If you need authentication you can add an additional line below the scoped registry with `_auth` and the basic authentication string for username and password (`username:password` ⇒ base64 encoded) or `_authToken` for an access token:
+
+`//<your registry>:_auth=<base64-string>`
+
+## What is this all about?
+
+Nowadays almost every programming language comes with its own package management solution, so developers can easily share packages with other developers around the world. For Python it is pip, for .NET it`s nuget. npm is the package manager solution for Javascript based languages. It is used to create and use node packaged modules and is built into the Javascript platform [Node.js](http://www.nodejs.org/). The central component behind these package managers is a registry. A registry is a database of packages, each comprised of software and metadata. For example the public registry for npm is [registry.npmjs.org](http://registry.npmjs.org).
```

Comment:
You are using the wrong apostrophe here:

it`s nuget -> it's nuget

Reply:
:+1: